### ISSUE TITLE
Scheduling option: `:greedy`

### ISSUE THREAD

**Issue by carstenbauer** [JuliaFolds2/OhMyThreads.jl#11]
Maybe we want to add something like https://github.com/JuliaLang/julia/pull/52096 does for `@threads`.

**Comment by MasonProtter** [comment]
Yeah that'd be nice to have. I think we could pretty much just wrap `Threads.foreach` to do this if I understand correctly.

**Comment by MasonProtter** [comment]
Implemented in #17
